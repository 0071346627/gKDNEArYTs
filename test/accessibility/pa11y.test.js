import pa11y from 'pa11y'

describe('Pa11y', () => {
  let results

  beforeAll(async () => {
    results = await pa11y('http://localhost:3000')
  })

  test('no accessibility issues detected by pa11y', () => {
    expect(results.issues.length).toEqual(0)
    expect(results).toMatchSnapshot()
  })
})
