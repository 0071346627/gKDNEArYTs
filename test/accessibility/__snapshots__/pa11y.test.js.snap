// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pa11y no accessibility issues detected by pa11y 1`] = `
Object {
  "documentTitle": "bbc",
  "issues": Array [],
  "pageUrl": "http://localhost:3000/weather/london",
}
`;
